### PR TITLE
validate task title

### PR DESCRIPTION
**Pull request by mim24sd**
In this pull request I will manage the validation of task title

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -47,25 +21,29 @@ closeNavButton.addEventListener("click", () => {
...
 addTaskButton.addEventListener("click", () => {
-  tasks.push({
-    id: tasks.length + 1,
-    title: taskTitleInput.value,
-    isDone: false,
-    createdAt: new Date().toISOString(),
-  });
-  console.log(tasks);
-  renderTasks(tasks);
+  if (Number(taskTitleInput.value) == 0) {
```
Comment (blacksrc) [inline index.js]:
```js
input.value.trim() === ""
```

File: index.js [hunk 2]
```diff
@@ -47,25 +21,29 @@ closeNavButton.addEventListener("click", () => {
...
+    alert("Task title input can not be empty!");
+  } else {
+    console.log;
+    if (taskTitleInput.value.replace(/\s/g, "").length < 3) {
```
Comment (blacksrc) [inline index.js]: Use `trim()` instead of regex

File: index.js [hunk 3]
```diff
@@ -47,25 +21,29 @@ closeNavButton.addEventListener("click", () => {
...
-  tasks.push({
-    id: tasks.length + 1,
-    title: taskTitleInput.value,
-    isDone: false,
-    createdAt: new Date().toISOString(),
-  });
-  console.log(tasks);
-  renderTasks(tasks);
+  if (Number(taskTitleInput.value) == 0) {
+    alert("Task title input can not be empty!");
```
Comment (blacksrc) [inline index.js]: Show this message under the input with red color instead of using alert.

File: index.js [hunk 4]
```diff
@@ -47,25 +21,29 @@ closeNavButton.addEventListener("click", () => {
...
+  } else {
+    console.log;
+    if (taskTitleInput.value.replace(/\s/g, "").length < 3) {
+      alert("Task title is too short!");
```
Comment (blacksrc) [inline index.js]: Same as the other one.

File: index.js [hunk 5]
```diff
@@ -47,25 +21,29 @@ closeNavButton.addEventListener("click", () => {
...
-    title: taskTitleInput.value,
-    isDone: false,
-    createdAt: new Date().toISOString(),
-  });
-  console.log(tasks);
-  renderTasks(tasks);
+  if (Number(taskTitleInput.value) == 0) {
+    alert("Task title input can not be empty!");
+  } else {
+    console.log;
```
Comment (blacksrc) [inline index.js]: ?

File: index.js (hunk 1 above)
Comment (mim24sd) [reply index.js]: Oh thanks fixed it ^^

File: index.js (hunk 2 above)
Comment (mim24sd) [reply index.js]: OK

File: index.js (hunk 5 above)
Comment (mim24sd) [reply index.js]: No word to say :/

File: index.js (hunk 3 above)
Comment (mim24sd) [reply index.js]: Got it

File: index.js (hunk 4 above)
Comment (mim24sd) [reply index.js]: OK

File: assests/javaScript/index.js [hunk 6]
```diff
@@ -47,25 +22,32 @@ closeNavButton.addEventListener("click", () => {
...
+
+  if (taskTitle === "") {
+    titleError.innerHTML = "Task title can not be empty!";
+    titleError.classList.add("title-error");
+  } else if (taskTitle.length < 3) {
+    titleError.innerHTML = "Task title is too short!";
+    titleError.classList.add("title-error");
+  } else {
+    titleError.innerHTML = "";
+
```
Comment (blacksrc) [inline assests/javaScript/index.js:36]: Add this as well.
```js
titleError.classList.remove("title-error");
```

Comment (mim24sd) [reply assests/javaScript/index.js:36]: fixed ^^

**Pull request merged**